x-pack/plugins/code/server/indexer/lsp_indexer.ts

```diff
@@ -180,7 +180,7 @@ export class LspIndexer extends AbstractIndexer {
         reference: this.options.enableGlobalReference,
       });
 
-      if (response && response.result.length > 0) {
+      if (response && response.result.length > 0 && response.result[0]) {
```

Comment:
should check `response.result`'s existence too

Reply:
good catch